inventory/sample/group_vars/k8s-cluster.yml

```diff
@@ -251,3 +251,16 @@ persistent_volumes_enabled: false
 ## See https://github.com/kubernetes-incubator/kubespray/issues/2141
 ## Set this variable to true to get rid of this issue
 volume_cross_zone_attachment: false
+
+## Container Engine Acceleration
+## Enable container accelartion feature, for example use gpu acceleration in containers
+container_engine_acceleration_enabled: false
```

Comment:
The default is already false so I think the sample should be
```
## Uncomment to enable GPU acceleration:
# container_engine_acceleration_enabled: true
```

Reply:
ok will be in next commit